src/main/java/entities/Engine.java

```diff
@@ -0,0 +1,52 @@
+package entities;
+
+import enums.TypeFuel;
```

Comment:
What do you think about using blank line between *custom* and *system*  imports?

Reply:
nice idea,